src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx

```diff
@@ -336,7 +336,10 @@ export const ArtworkSidebarCommercialButtons: React.FC<ArtworkSidebarCommercialB
     makeOffer?: ResponsiveValue<"primaryBlack" | "secondaryBlack">
     contactGallery?: ResponsiveValue<"primaryBlack" | "secondaryBlack">
   } = {}
-  if (artwork.isAcquireable || activePartnerOffer) {
+  if (
+    artwork.isAcquireable ||
+    (activePartnerOffer && activePartnerOffer.isAvailable)
```

Comment:
My only concern here is that our definitions align with how we test for showing the unavailable banner when an artwork page expects a partner offer: See [these lines from 13722](https://github.com/artsy/force/pull/13722/files#diff-8dfda62ff39e008f30d73abc8aaa0ee90fbfe4edfb36092f6c36bcb1bff3c745R66-R74) (uses `artwork.isPurchaseable`) and [this slack thread](https://artsy.slack.com/archives/C02JHHHKP5K/p1712782539452009) for some work I did there:
> isAcquirable is for buy now
isPurchasable is inventory + availability == for_sale + merchant account configured + price listed in supported commerce currency
isSold is availability == sold

☝️ These all refer to the artwork. Meanwhile `PartnerOffer#available` is `partnerOffer.artwork.purchasable` so it seems good (more names aside).

Reply:
Could probably be shortened to `activePartnerOffer?.isAvailable`